atomic_reactor/plugins/pre_add_content_sets.py

```diff
@@ -55,7 +55,9 @@ def get_output_json(self):
         file_path = os.path.join(workdir, REPO_CONTENT_SETS_CONFIG)
         content_sets = {}
 
-        if os.path.exists(file_path):
+        config_data = self.workflow.source.config.compose or {}
+
+        if os.path.exists(file_path) and config_data.get('pulp_repos'):
```

Comment:
`content_sets.yml` serve multiple purposes. One of them is to allow ODCS integration via the `compose.pulp_repos` flag in `container.yaml`. The second one is to aid in grading. So although ODCS integration may not be enabled, we still want to add the content sets to the image.

Reply:
ok I'll close this